internal/jimm/auditlog/auditlog_test.go

```diff
@@ -72,29 +74,85 @@ func (s *auditLogManagerSuite) TestFindAuditEvents(c *qt.C) {
 
 	now := (time.Time{}).UTC().Round(time.Millisecond)
 
+	params := map[string]string{
+		"key": "value",
+	}
+	sensitive_params := map[string]string{
+		"sensitive_data": "secret_value",
+	}
+
+	sensitive_data, err := json.Marshal(sensitive_params)
+	c.Assert(err, qt.IsNil)
+	data, err := json.Marshal(params)
+	c.Assert(err, qt.IsNil)
+
 	events := []dbmodel.AuditLogEntry{{
 		Time:         now,
 		IdentityTag:  s.adminUser.Identity.Tag().String(),
 		FacadeMethod: "Login",
+		Params:       sensitive_data,
 	}, {
 		Time:         now.Add(time.Hour),
 		IdentityTag:  s.adminUser.Identity.Tag().String(),
 		FacadeMethod: "AddModel",
+		Params:       data,
 	}, {
 		Time:         now.Add(2 * time.Hour),
 		IdentityTag:  s.priveligedUser.Identity.Tag().String(),
 		Model:        "TestModel",
 		FacadeMethod: "Deploy",
+		Params:       data,
 	}, {
 		Time:         now.Add(3 * time.Hour),
 		IdentityTag:  s.priveligedUser.Identity.Tag().String(),
 		Model:        "TestModel",
 		FacadeMethod: "DestroyModel",
+		Params:       data,
+	}, {
+		Time:         now.Add(4 * time.Hour),
+		IdentityTag:  s.adminUser.Identity.Tag().String(),
+		FacadeMethod: "LoginWithSessionToken",
```

Comment:
Don't you think these tests are a bit pointless? We're just testing all these value exists in a list. We can just test the redact method does the expected thing, in a separate test possibly.

Reply:
done